azure-kusto-data/azure/kusto/data/_version.py

```diff
@@ -1,3 +1,3 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License
-VERSION = "3.1.0"
+VERSION = "3.1.1"
```

Comment:
I see 2 other commits merged before this which aren't yet released. With that in mind, wouldn't it better to separate this version bump into its own PR?